modules/apps/object/object-test/src/testFunctional/tests/ObjectSalesforce.testcase

```diff
@@ -21,4 +21,84 @@ definition {
 		}
 	}
 
+	@description = "LPS-185427 - Verify CRUD functionality with an imported custom object using Salesforce as the storage type."
+	@priority = 5
+	test AssertCRUDWithImportedCustomObject {
+		property portal.acceptance = "true";
+
+		task ("Given portal configured with Salesforce") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Configuration",
+				panel = "Control Panel",
+				portlet = "Instance Settings");
+
+			SystemSettings.gotoConfiguration(
```

Comment:
I thought you were going to use instance settings? If we're using system settings, we need to skip the virtual instance.